src/containers/Login/components/AddProfileDetails.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import UserSettings from 'containers/Settings/UserSettings';
+import { Modal, Text, Button } from 'components';
+import { PageCard } from 'explorer-components';
+import { actions } from '../reducer';
+
+const AddProfileDetails = props => {
+  const { visible, onClose, showForm, form } = props;
+
+  if (form) {
+    return (
+      <Modal visible={visible} size="large" dismissable onClose={onClose}>
+        <Modal.Header closable icon={['fal', 'faAddressCard']}>
+          <Modal.Heading>Profile Details</Modal.Heading>
+        </Modal.Header>
+        <Modal.Body>
+          <div className="container-fluid">
+            <div className="row center-xs">
+              <div className="col-xs-5">
+                Add some profile details about yourself, so that others on the
+                platform have an idea of your background and skillset, and can
+                find you elsewhere online.
+                <br />
+                <br />
+                We recommend you fill out as many details as possible, since
+                you'll need to submit a small transaction each time you change
+                in order for them to be kept on-chain.
+                <br />
+                <br />
+                FAQ: Why are my profile details kept on-chain
+              </div>
+            </div>
+          </div>
+
+          <PageCard.Break />
+          <UserSettings />
+        </Modal.Body>
+      </Modal>
+    );
+  }
+
+  return (
+    <Modal visible={visible} size="small" dismissable onClose={onClose}>
+      <Modal.Header closable icon={['fal', 'idCardAlt']}>
+        <Modal.Heading>Profile Details</Modal.Heading>
+      </Modal.Header>
+      <Modal.Body>
+        <Text>
+          Welcome to the Bounties Network! We're excited to see what you bounty.
+          <br />
+          <br />
+          Before you get started, we encourage you to add some profile details
+          about yourself, so that others on the platform have an idea of your
+          background and skillset.
+          <br />
+          <br />
+          In order to add details to your profile, you'll be required to approve
+          a small transaction. This is so we can keep your profile information
+          securely on-chain.
+          <br />
+          <br />
+          FAQ: Why are my profile details kept on-chain?
+        </Text>
+      </Modal.Body>
+      <Modal.Footer>
+        <Button type="" onClick={onClose}>
```

Comment:
add the margin prop here... ` <Button type="" onClick={onClose} margin>`
And then it will look beautiful :)